File: Application/Services/Operations/BenchBudgetService/Dtos/BudgetServiceDto.cs
using System;
using Application.Services.Operations.Main.Companies.Dtos;
using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.BenchBudgetService.Dtos.Enums;
using Application.Services.Operations.Authentication.Dtos;

namespace Application.Services.Operations.BenchBudgetService.Dtos
{
    public class BudgetServiceDto
    {
        public int Id { get; set; }
        public int CompanyId { get; set; }
        public CompanyDto Company { get; set; }
        public int UserId { get; set; }
        public MyUserDto User { get; set; }
        public int CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public string ProblemAccordingCustomer { get; set; }
        public string IsPresentVisuallyDescription { get; set; }
        // public bool IsRemote { get; set; }
        public ExecutionModeEnumDto ExecutionMode { get; set; }
        public string DataDescription { get; set; }
        public DateTime EntryDate { get; set; }
        // public DateTime BudgetOpen { get; set; }
        public ServiceDto Service { get; set; }
        public CollectDeliverCostsDto CollectsDeliversCosts { get; set; }
        public StatusServiceEnumDto StatusService { get; set; }
        // public List<TrackingDto> HardwareIncludedInServices { get; set; }
    }
}

File: Views/src/components/budget-service-bench/dto/budget-Service-dto.ts
import { MyUser } from "src/components/authentication/dto/myUser";
import { CustomerDto } from "src/components/main/customer/dtos/customer-dto";
import { CompanyDto } from "src/shared/dtos/company-dto";
import { ServiceDto } from "./Service-dto";
import { StatusServiceEnumDto } from "./enums/status-service-enum-dto";
import { CollectDeliverCostsDto } from "./collect-deliver-costs-dto";

export class BudgetServiceDto
{
    id:number ;
    companyId:number ;
    company:CompanyDto ;
    userId:number ;
    user:MyUser ;
    customerId:number ;
    customer:CustomerDto ;
    problemAccordingCustomer:string ;
    isPresentVisuallyDescription:string ;
    isRemote:boolean ;
    dataDescription:string ;
    entryDate:Date ;
    budgetOpen:Date ;
    service:ServiceDto ;
    collectsDeliversCosts:CollectDeliverCostsDto ;
    statusService:StatusServiceEnumDto ;
}
